scripts/train_causal.py

```diff
@@ -219,12 +218,13 @@ def peft_model(model):
     requires some care. The state_dict will also contain all paramters, not
     just the adapter, though we plan to use merged tuned models for now.
     """
-    from fms.models.hf.llama.modeling_llama_hf import HFAdaptedLLaMAForCausalLM
-
-    model = to_hf_api(model)
     from peft import LoraConfig, get_peft_config, get_peft_model
     from peft.mapping import PeftModelForCausalLM
 
+    from fms.models.hf.llama.modeling_llama_hf import HFAdaptedLLaMAForCausalLM
```

Comment:
is this needed? I don't see it being used.

Reply:
hmm...I don't see it used but thinking it must have been there for a reason. Does importing it register the set of HF adapters available, or anything like that?

Reply:
I think `to_hf_api` will handle everything, the adapters are getting registered in fms.models.hf (__init__.py)